docs/PremiumApps.md

```diff
@@ -0,0 +1,369 @@
+# Premium Apps Installation Guide
+
+The Splunk Operator automates the installation of Enterprise Security with support for other premium apps coming in the future releases. This page documents the prerequisites, installation steps, troublshooting steps, and limitations of deploying premium apps using the Splunk Operator.
+
+## Enterprise Security
+
+### Prerequisites
+
+Installing Enterprise Security in a Kubernetes cluster with the Splunk Operator requires the following:
+
+* Ability to utilize the Splunk Operator [app framework](https://splunk.github.io/splunk-operator/AppFramework.html) method of installation.
+* Access to the [Splunk Enterprise Security (ES)](https://splunkbase.splunk.com/app/263/) app package.
+* Splunk Enterprise Security version 7.1.0, 7.0.2, 7.0.1, 7.0.0 or 6.6.2 as ES support in Splunk Operator is starting from Splunk Operator Release 2.2 which requires Splunk Enterprise 9.0.3 or later. For more information regarding Splunk Enterprise and Enterprise Security compatibility, see the [version compatibility matrix](https://docs.splunk.com/Documentation/VersionCompatibility/current/Matrix/CompatMatrix).
```

Comment:
nit: starting from Splunk Operator Release 2.2.0...
Also, we will have to change the version from 9.0.3 to 9.0.3-a1 or something like that once we merge the ansible PR.

Reply:
2.2.0 change is done.
9.0.3-a1 related change remaining